clients/venice-admin-tool/src/main/java/com/linkedin/venice/SystemStorePushTask.java

```diff
@@ -0,0 +1,168 @@
+package com.linkedin.venice;
+
+import static com.linkedin.venice.AdminTool.printObject;
+
+import com.linkedin.venice.common.VeniceSystemStoreType;
+import com.linkedin.venice.controllerapi.ControllerClient;
+import com.linkedin.venice.controllerapi.ControllerResponse;
+import com.linkedin.venice.controllerapi.JobStatusQueryResponse;
+import com.linkedin.venice.controllerapi.StoreResponse;
+import com.linkedin.venice.controllerapi.UpdateStoreQueryParams;
+import com.linkedin.venice.controllerapi.VersionCreationResponse;
+import com.linkedin.venice.controllerapi.VersionResponse;
+import com.linkedin.venice.meta.Version;
+import com.linkedin.venice.pushmonitor.ExecutionStatus;
+import com.linkedin.venice.utils.Utils;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+/**
+ * This class aims to do one time emtpy push to all user system stores of a specific user store.
+ * It will aggregate and compute the largest used version from all regions and update store before performing empty push.
+ * It will also skip empty push to store which is being migrated and is in the destination cluster.
+ */
+public class SystemStorePushTask implements Function<String, Boolean> {
+  public static final String TASK_NAME = "PushSystemStore";
+  private static final Logger LOGGER = LogManager.getLogger(SystemStorePushTask.class);
+  private static final int JOB_POLLING_RETRY_COUNT = 200;
+  private static final int JOB_POLLING_RETRY_PERIOD_IN_SECONDS = 5;
+  private static final String SYSTEM_STORE_PUSH_TASK_LOG_PREFIX = "[**** SYSTEM STORE PUSH ****]";
+  private static final List<VeniceSystemStoreType> SYSTEM_STORE_TYPE =
```

Comment:
I think it's useful to provide this list as an input so we can choose to do system store empty push just for meta, dvc push status or both.